platformsh-laravel-env.php

```diff
@@ -136,3 +135,11 @@ function mapPlatformShRedisSession(string $relationshipName, Config $config) : v
     setEnvVar('REDIS_HOST', $credentials['host']);
     setEnvVar('REDIS_PORT', $credentials['port']);
 }
+
+
+function mapPlatformShMail(string $relationshipName, Config $config) : void
+{
+    setEnvVar('MAIL_DRIVER', 'smtp');
+    setEnvVar('MAIL_HOST', $config->smtpHost);
+    setEnvVar('MAIL_PORT', '587');
```

Comment:
this is also the default in Laravel already .. the port doesn't seem to be exposed by the platforn.sh config class